apps/s_server.c

```diff
@@ -2208,6 +2214,9 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
         goto err;
     }
 
+    if (s_force_pha)
+        SSL_force_post_handshake_auth(con);
```

Comment:
What does this actually do on the server side? The docs seem to indicate its a client side option.

Reply:
I added it to the server-side to permit testing; allows the server to initiate a post-handshake auth without receiving the extension.